### PR TITLE
Fix table.describe

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: python/hail/table.py [hunk 1]
```diff
@@ -1413,31 +1413,33 @@ def collect(self):
         """
         return TArray(self.schema)._convert_to_py(self._jt.collect())
 
-    @typecheck_method(truncate_at=integral)
     def describe(self, truncate_at=60):
-        """Print information about the fields in the table."""
+        """Print information about the fields in the matrix."""
```
Comment (konradjk) [inline python/hail/table.py]: undo this line?

Comment (patrick-schultz) [inline python/hail/table.py]: Why matrix?

File: python/hail/table.py [hunk 2]
```diff
@@ -1413,31 +1413,33 @@ def collect(self):
         """
         return TArray(self.schema)._convert_to_py(self._jt.collect())
 
-    @typecheck_method(truncate_at=integral)
     def describe(self, truncate_at=60):
```
Comment (patrick-schultz) [inline python/hail/table.py]: `truncate_at` isn't being used, correct?

Comment (tpoterba) [reply python/hail/table.py]: yup, will remove.

**Pull request merged**